internal/pkg/service/common/configmap/viper.go

```diff
@@ -0,0 +1,35 @@
+package configmap
+
+import (
+	"github.com/keboola/go-utils/pkg/orderedmap"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/env"
+)
+
+// BindToViper flags, ENVs and config files to the Viper configuration registry.
+// It is recommended to use the Bind instead of the BindToViper method if you have a choice,
+// as the configuration structure is easier to use, both in production code and tests.
+func BindToViper(viper *viper.Viper, flags *pflag.FlagSet, flagToField FlagToFieldFn, envs env.Provider, envNaming *env.NamingConvention, configFiles []string) (map[string]SetBy, error) {
```

Comment:
V CLI este pouzivame `VIper`, takze pre spatnu kompatibilitu tu pridavam `BindToViper`.

Rozdiel je v tom, ze Viper sa pouziva nejak takto:
```
v.GetBool("verbose")
```

A konfiguracna struktura sa pouziva takto:
```
config.Verbose
```

Teda jednoduchsie sa to refactoruje, nemusis vediet typ hodnoty.

----------

Viper ma tiez metodu `Unmarshal(struct)`, ktora namapuje hodnoty na strukturu.
Ale boli s tym problemy, takzko sa ten mapping upravoval.

Preto `Bind` metoda nepouziva `Viper` vobec.